frontend/public/components/custom-resource-definition.tsx

```diff
@@ -107,6 +114,53 @@ const Established: React.FC<{ crd: CustomResourceDefinitionKind }> = ({ crd }) =
   );
 };
 
+const EmptyVersionsMsg: React.FC<{}> = () => <EmptyBox label="CRD Versions" />;
+
+const CRDVersionsHeader = () => [
+  {
+    title: 'Name',
+    sortField: 'name',
+    transforms: [sortable],
+  },
+  {
+    title: 'Served',
+    sortField: 'served',
+    transforms: [sortable],
+  },
+  {
+    title: 'Storage',
+    sortField: 'storage',
+    transforms: [sortable],
+  },
+];
+
+const CRDVersionsRows = ({ componentProps: { data } }) =>
+  _.map(data, (version: CRDVersion) => [
+    {
+      title: version.name,
+    },
+    {
+      title: version.served.toString(),
+    },
+    {
+      title: version.storage.toString(),
+    },
+  ]);
+
+const CRDVersionList: React.FC<CRDVersionProps> = ({ crdVersions }) => {
+  return (
+    <Table
```

Comment:
We might use the PF `Table` directly instead of the console `Table`. Console `Table` is really meant for resource list views, and  PF `Table` would also let us sort the version column with your custom sort.